docs/index.rst

```diff
@@ -6,6 +6,12 @@
 ctwrap - Python Wrapper for Batch Simulations
 =============================================
 
+.. include:: ../README.rst
+
+.. include:: ../install.rst
+
+.. include:: ../CONTRIBUTING.rst
```

Comment:
I'd prefer to use dedicated pages for these, e.g. see `camsym` for installation. That way the page remains at a manageable length. The `install.rst` should probably be `INSTALL.rst` in the root folder. For whatever reason, it appears to be custom to do so.

Reply:
Please address / also see other comment.